build.sbt

```diff
@@ -47,9 +47,9 @@ lazy val scala213CompilerOptions = Seq(
 
 val testDependencies = Seq(
   "org.scalacheck" %% "scalacheck" % "1.14.2",
-  "org.scalatest" %% "scalatest" % "3.1.2",
+  "org.scalatest" %% "scalatest" % "3.1.4",
   "org.typelevel" %% "cats-laws" % catsVersion,
-  "org.typelevel" %% "discipline-scalatest" % "1.0.0"
+  "org.typelevel" %% "discipline-scalatest" % "1.0.1"
```

Comment:
there is [2.0.0](https://github.com/typelevel/discipline-scalatest/releases)

Reply:
fixed